experimental/rel_component/configs/rel_tok2vec.cfg

```
@@ -0,0 +1,114 @@
+[paths]
+train = null
+dev = null
+raw = null
+init_tok2vec = null
+
+[system]
+seed = 342
+gpu_allocator = null
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec", "relation_extractor"]
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.tok2vec]
+factory = "tok2vec"
+
+[components.tok2vec.model]
+@architectures = "spacy.HashEmbedCNN.v1"
+pretrained_vectors = null
+width = 96
+depth = 2
```

```suggestion
depth = 2
```

If you haven't tried 4 here, do try that.

---

That didn't improve things, though that may be because the dataset is still quite small.